src/primitives/bezier.jl

```diff
@@ -0,0 +1,89 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    BezierCurve(points)
+
+A Bézier curve with control points `points`.
+See https://en.wikipedia.org/wiki/Bézier_curve.
```

Comment:
From the Wikipedia article, what is the type of Bezier curve you implemented? Is it of a fixed order (linear, quadratic, cubic) or a recursive definition? I need to learn about these curves more seriously at some point. They are really nice.

Reply:
It's a recursive definition, for both evaluation algorithms. If you have a curve with 3 points it's quadratic, 4 it's cubic, and so on.

Reply:
I would state this explicitly in the doc string then: `A recursive Bezier curve with control points`.